### ISSUE TITLE
sqlparser.cpp missing from repo

### ISSUE THREAD

**Issue by ysalmon** [rparkins999/sqliteman#1]
Hello,
I am trying to build your version of sqliteman. I am encountering difficulties with sqlparser.cpp. This file is not present in HEAD; in fact git log ./sqlparser.cpp gives no result, so I think it was never added to the repo.
At first I thought that sqlparser.cpp was an old thing replaced with sqlparser/tosqlparse.cpp and began modifying files referencing sqlparser.cpp accordingly.

But this does not work and sqlparser.cpp seems to be an addition by you: references to it were added by commit dfc3ad0d9db7f7078e01c1fe43c8948acde0d176 New much faster schema parser not using regexp's on October the 26th.

Did you forget sqlparser.cpp in this commit ?
Is sqlparser/tosqlparse.cpp still used ? Or are these two files implementing different things ?

**Comment by ysalmon** [comment]
Thanks for your quick reaction.
It seems that ui_createviewdialog.h is also missing from the repo.

**Comment by rparkins999** [comment]
Ah, that's a trickier problem. It's a built file, so it shouldn't be in the repo at all. The issue here that CreateViewDialog no longer has its own UI; I changed it some while ago so it's built on the Ui of TableEditorDialog, and removed createviewdialog.ui from the repo. I hadn't noticed that AlterViewDialog uses it. Since the old built version of ui_createviewdialog.h is still there in my working directory, it was building without problems for me.
I've just pushed alterviewdialog.ui (which is in fact a copy of the old createviewdialog.ui) and the changes necessary to make AlterViewDialog build with it.
Richard
